cibuildwheel/resources/pyodide_python.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/sh
```

Comment:
Can't download it from the Pyodide repo?

Reply:
This is a modified version of the script. The modification is in pyodide main branch but I failed to backport it into 0.23.1. I will release 0.23.2 with this and a couple of other changes and then remove this file.

Reply:
Okay fixed now.

Reply:
Fixed now.